workspaces/jupyterlab-python/install.sh

```diff
@@ -2,18 +2,42 @@
 
 set -ex
 
+# This script requires root.
+[ "$(id -u)" -eq 0 ] || exit 1
+
```

Comment:
Nit: is this necessary? I don't see this in other workspaces and since this is always called from the Dockerfile with `USER root` preceding it, I don't see how this could end up being run as non-root.

Reply:
It's just an assertion to avoid configuration errors. It could be removed (most of the following script would fail if the condition isn't met anyway).

Reply:
I'll leave that call to you; approval stands.

Reply:
All right, I went ahead and removed it.